scripts/slack_notification.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import warnings
+
+import slack
+from slack.errors import SlackApiError
+
+
+def post_slack_notification():
```

Comment:
Minor, but it would be cleaner if we separated out the parsing of environment variables into a separate method.